app/styles/globals.css

```diff
@@ -18,3 +25,11 @@ a {
 * {
   box-sizing: border-box;
 }
+
+.wallet-adapter-button {
+  border-radius: .75rem;
+}
+
+button:hover {
+  background-color: #374050 !important;
```

Comment:
this is global, so if we had buttons with different colors, this would make them all the same hover color. 

We should try to stick to using DaisyUI/TailwindCSS for our styling needs, so you can check out how they do hover styles: https://tailwindcss.com/docs/hover-focus-and-other-states